source/includes/_roles.md

```diff
@@ -0,0 +1,270 @@
+# Roles
+## List Roles
+
+```javascript
+fetch('https://core.eventtia.com/v1/roles/', {
+  method: 'GET',
+  headers: {
+    'Authorization': '<your token>',
+  }
+})
+```
+> Make sure you replace &lt;your token&gt; with the JWT you get when you authenticate. 
+
+> Example of a successful (200) response:
+
+```http
+HTTP/1.1 200 OK
+{
+  "data": [
+    {
+      "id": "1",
+      "type": "roles",
+      "attributes": {
+        "id": 1,
+        "name": "admin"
+      }
+    }
+  ],
+  "links": {
+    "first": "http://localhost:3000/api/v1/es/v1/roles?page[number]=1",
+    "last": "http://localhost:3000/api/v1/es/v1/roles?page[number]=1",
+    "prev": null,
+    "next": null
+  }
+}
+```
+
+This endpoint return a list of roles
```

Comment:
This endpoint returns a list of roles that your account have configured

Reply:
done